src/Hedgehog.Experimental.CSharp.Tests/GenTests.cs

```diff
@@ -0,0 +1,90 @@
+namespace Hedgehog.Experimental.CSharp.Tests;
+
+using Hedgehog.Linq;
+using static Hedgehog.Linq.Property;
+using FluentAssertions;
```

Comment:
Visual Studio 2022 (Community Edition) has nine messages about how to improve this code.  Three of them are to move these `using` declarations before the `namespace`.  I tried to do that before but ran into the problems (as mentioned before/elsewhere).

Can you address these nine messages?

Maybe it would help if this `namespace` were `Hedgehog.Linq.Tests`.

Reply:
Unlikely. I don't have Visual Studio and I don't run (or have access to) Windows.